python/triton/language/core.py

```diff
@@ -378,7 +378,7 @@ def __init__(self, name):
                 self.primitive_bitwidth = 32
                 self.exponent_bias = 127
             elif name == 'fp64':
-                self.fp_mantissa_width = 53
+                self.fp_mantissa_width = 52
```

Comment:
did you mean to change this? Seems unrelated to the PR message

Reply:
oops I stacked my commits on the same branch by mistake